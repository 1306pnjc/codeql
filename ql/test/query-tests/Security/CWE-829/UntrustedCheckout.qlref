Security/CWE-829/UntrustedCheckout.ql
