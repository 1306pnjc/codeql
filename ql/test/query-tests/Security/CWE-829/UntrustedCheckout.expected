| .github/workflows/auto_ci.yml:20:9:27:6 | Uses Step | Potential unsafe checkout of untrusted pull request on privileged workflow. |
| .github/workflows/auto_ci.yml:67:9:74:6 | Uses Step | Potential unsafe checkout of untrusted pull request on privileged workflow. |
| .github/workflows/gitcheckout.yml:10:11:18:8 | Run Step | Potential unsafe checkout of untrusted pull request on privileged workflow. |
| .github/workflows/untrusted_checkout.yml:9:7:13:4 | Uses Step | Potential unsafe checkout of untrusted pull request on privileged workflow. |
